src/main/java/frc/robot/subsystems/swerve/SwerveSubsystem.java

```diff
@@ -121,6 +125,20 @@ public void periodic() {
         );
     }
 
+    /**
+     * Toggles whether the subsystem is locked (whether control input has no effect and the wheels 
+     * should lock themselves to prevent shoving).
+     */
+    public void toggleLocked() {
```

Comment:
I think our software design is intended to avoid actually interfacing with motors as a result of button presses, but rather changing states and allowing each module to interface with its motors as part of its periodic loop. 

so I think this motor setting logic should go in periodic

Reply:
another thought is that the correct logic might not be to immediately turn the modules. we might want to put in logic to lock only after a certain delay, or after the robot is going at less than a certain velocity, even if we dont use it?

i know in FTC, the one team that built a swerve robot could immediately lock their wheels and when they go full speed and let go of the joystick they nearly flip over